package.json

```diff
@@ -27,15 +27,18 @@
   },
   "dependencies": {
     "@azure/msal-node": "^1.1.0",
+    "@types/ioredis": "^4.26.4",
     "axios": "^0.21.1",
     "cookie": "^0.4.1",
     "google-libphonenumber": "^3.2.20",
     "govuk-frontend": "^3.11.0",
+    "ioredis": "^4.27.3",
     "next": "10.2.0",
     "nextjs-basic-auth-middleware": "^0.2.0",
     "notifications-node-client": "^5.1.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "redis-json": "^6.0.0",
```

Comment:
Necessary to store JSON objects in Redis.  See blog post at [redislabs.com](https://redislabs.com/blog/redis-as-a-json-store/).  The `redis-json` package is a wrapper for the RedisJSON module.